stats/big-simulation.js

```diff
@@ -0,0 +1,37 @@
+let Simulation = require('./simulation')
+let modules = require('./strategies')
+let feeRate = 56 * 100
+let results = []
+
+let list = require('./moneypot-hotwallet.json')
+
+// for each strategy
+for (var name in modules) {
+  let f = modules[name]
+  let simulation = new Simulation(name, f, feeRate)
+
+  console.log(name)
+  let i = 0
+  for (var value of list) {
+    i++
+
+    if (i % 1000 === 0) {
+      console.log(i)
+    }
+    let txo = {
+      address: 'A',
+      value: Math.abs(value)
```

Comment:
Frankly - I am a little stuck on the `{script: {length}}` now.

It is used both in utxos and txos in simulation.js; however, it is also used in utils.outputbytes for both inputs and outputs! So both input size and output script sizes are the same in the simulations, which makes the results wrong :)